funnel/models/event.py

```diff
@@ -125,6 +134,24 @@ class Participant(BaseMixin, db.Model):
     __table_args__ = (db.UniqueConstraint('proposal_space_id', 'email'),)
 
     @classmethod
+    def get(cls, space, email, participant_attrs={}, create=False):
+        participant = cls.query.filter_by(proposal_space=space, email=email).first()
+        if create and not participant:
+            participant = cls(
```

Comment:
What happens if you call with `create=True` and the record already exists? `participant_attrs` is ignored then, even if the data doesn't match? That makes this method's purpose and behaviour confusing.